docs/cpp/multiple-base-classes.md

```diff
@@ -157,7 +156,7 @@ pc->B::a();
 
 ### Dominance
 
-it's possible for more than one name (function, object, or enumerator) to be reached through an inheritance graph. Such cases are considered ambiguous with nonvirtual base classes. They're also ambiguous with virtual base classes, unless one of the names "dominates" the others.
+It's possible for more than one name (function, object, or enumerator) to be reached through an inheritance graph. Such cases are considered ambiguous with nonvirtual base classes. They're also ambiguous with virtual base classes, unless one of the names "dominates" the others.
```

Comment:
Accidentally left out this nit yesterday (added in commit https://github.com/MicrosoftDocs/cpp-docs/commit/e397c593e2629bfbe010680f8e8f4c9ce53a8f16).

Reply:
Is it just me or is it ironic that the sentence under 'dominance' starts out lowercase ;-)